src/test/java/testQaSoCalVictoria.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class testQaSoCalVictoria {
+    @Test
+    public void itemInCard() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\vilun\\Downloads\\chromedriver_win32\\chromedriver.exe");
+
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.saucedemo.com/");
+
+        WebElement userName = driver.findElement(By.id("user-name"));
+        WebElement password = driver.findElement(By.id("password"));
+        WebElement loginBtn = driver.findElement(By.id("login-button"));
+
+        userName.sendKeys("standard_user");
+        password.sendKeys("secret_sauce");
+        loginBtn.click();
+
+        WebElement addToCartBtn = driver.findElement(By.id("add-to-cart-sauce-labs-backpack"));
+        addToCartBtn.click();

```

Comment:
If you use a variable just once, there is no point to create a variable
```suggestion
        driver.findElement(By.id("add-to-cart-sauce-labs-backpack")).click();

```

Reply:
changed